src/gbinder_driver.c

```diff
@@ -414,7 +414,7 @@ void
 gbinder_driver_handle_transaction(
     GBinderDriver* self,
     GBinderObjectRegistry* reg,
-    GBinderHandler* h,
+    GBinderHandler* handler,
```

Comment:
This looks like an unnecessary change

Reply:
`h` is not so obvious, in other functions same value has name `handler`